app/controllers/catalog_controller.rb

```diff
@@ -207,6 +207,10 @@ def show
 
     @cocina = maybe_load_cocina(params[:id])
     flash[:alert] = 'Warning: this object cannot currently be represented in the Cocina model.' if @cocina.instance_of?(NilModel)
+    if @document[SolrDocument::FIELD_WORKFLOW_ERRORS]
+      flash[:error] =
+        "Error: #{helpers.value_for_wf_error(document: @document, field: SolrDocument::FIELD_WORKFLOW_ERRORS)}"
+    end
 
```

Comment:
if Solr document contains workflow errors, use the existing helper method to add them to the error flash